README.md

```diff
@@ -23,38 +23,110 @@ Ansible Role for installing the Docker runtime environment on your (Debian) host
 
 ## Requirements
 
-*nothing special*
+This is intended for Debian GNU/Linux based systems.
```

Comment:
Please don't use a plain "this" at the beginning of a paragraph, as unnecessary ambiguity is introduced.